CourseGrab/Extensions/UserDefaults+Extensions.swift

```diff
@@ -28,6 +28,15 @@ extension UserDefaults {
         }
     }
     
+    var isLocalTimezoneEnabled: Bool {
```

Comment:
The only caveat I see is we don't reset the preference if the user signs out, so if someone else were to sign-in on the same device we would have the old preference. However, without us sending this to backend it doesn't seem like there is a way to account for this. If we do reset it upon sign out, each new user would simply have to re-toggle the preference. Either way, it is an extremely minor edge case so IMO fine to choose one option and leave as be for now. What do other people think?

Reply:
Not sure what the chances are for two people sharing a phone, but if we reset upon sign out, then a same person who signs back in won't get their preference saved.

Reply:
as a user I would expect my preference for timezone to be saved, so i would lean away from resetting it upon sign out. the most robust solution would be sending the info to backend (i also can't think of another way to account for it), but as yanlam pointed out it seems pretty unlikely that people would use the same phone -- tl:dr i think for now this approach is okay